source/code-generation/query-converter/convert-queries.txt

```diff
@@ -0,0 +1,161 @@
+.. _rm-convert-queries:
+
+===============
+Convert Queries
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can convert SQL queries to MongoDB syntax with query converter. 
+Query converter considers the mapping rules and schema 
+transformations defined in your project when converting your SQL code.
+
+About this Task
+---------------
+
+.. include:: /includes/fact-query-converter-generic.rst
+
+Before you Begin
+----------------
+
+.. include:: /includes/fact-query-converter-disclaimer.rst
+
+Steps
+-----
+
+.. procedure::
+   :style:  normal
+
+   .. step:: Navigate to the query converter pane
+
+      a. From the :guilabel:`Code Generation` tab, click the 
```

Comment:
[suggestion]

For steps that only have one substep, I'd suggest not enumerating them with `a.`. The lone substeps currently seem to provide additional context / instruction for the top-level step, so I don't really think of them as substeps.

```suggestion
      From the :guilabel:`Code Generation` tab, click the 
```